src/java/org/apache/cassandra/io/sstable/format/SSTableReader.java

```diff
@@ -2003,6 +2003,14 @@ public void addTo(Ref.IdentityCollection identities)
 
     }
 
+    public boolean isPresent(DecoratedKey key)
```

Comment:
maybe rename this to indicate that there are no guarantees here - or invert it to say `isNotPresent` ?